assets/javascripts/app/settings.coffee

```diff
@@ -104,3 +110,38 @@ class app.Settings
     @store.reset()
     @cache = {}
     return
+
+  initLayout: ->
+    @toggleDark(@get('dark'))
+    @toggleLayout(layout, @hasLayout(layout)) for layout in @LAYOUTS
+    @addResizerCSS()
+
+  toggleDark: (enable) ->
+    classList = document.documentElement.classList
+    classList[if enable then 'remove' else 'add']('_theme-default')
+    classList[if enable then 'add' else 'remove']('_theme-dark')
+
+  toggleLayout: (layout, enable) ->
+    classList = document.body.classList
+    classList[if enable then 'add' else 'remove'](layout) unless layout is SIDEBAR_HIDDEN_LAYOUT
+    classList[if $.overlayScrollbarsEnabled() then 'add' else 'remove']('_overlay-scrollbars')
+
+  addResizerCSS: ->
```

Comment:
Could this be replaced with CSS variables?

- Add the declarations below to their appropriate CSS files, reference `var(--sidebar-size)`
- Declare `--sidebar-size: 20rem` on `:root`
- Override it using `document.documentElement.style.setProperty('--sidebar-size', size + 'px')`